samples/certs/Makefile

```diff
@@ -121,3 +128,57 @@ root-key.pem:
 	@echo "O = $(CITADEL_ORG)" >> $@
 	@echo "CN = $(CITADEL_CN)" >> $@
 	@echo "L = $(L:/=)" >> $@
+
+#------------------------------------------------------------------------
+##<name>-certs-wl: generate certificates for a virtual machine connected to the namespace `$NAMESPACE` using serviceAccount `$SERVICE_ACCOUNT` as well as intermediate certificates for K8s cluster `$NAME`
+.PHONY: %-certs-wl
```

Comment:
What is 'wl' ?

Reply:
workload.

Reply:
And why do we need this abbreviation ? Is there anything else we would use certs for ? 

Let's keep it clean - NAMESPACE-certs is good enough.

Reply:
BTW - my point in the other comments was that we should be organized - the certs for a namespace 'foo' should be in a directory 'foo' - and the make target should be 'foo-certs'